backend/bamboo/models.py

```diff
@@ -0,0 +1,281 @@
+
+    site = relationship('Site', back_populates='pages')
+
+
+class Notification(db.Model):
+    __tablename__ = 'notification'
+    id = Column(Integer, primary_key=True)
+    content = Column(Text)
+    created_at = Column(DateTime, default=datetime.utcnow)
+    updated_at = Column(DateTime, default=datetime.utcnow, onupdate=datetime.utcnow)
+    site_id = Column(Integer, ForeignKey('site.id'))
+
+    site = relationship('Site', back_populates='notifications')
+
+
+class VolunteerForm(db.Model):
+    __tablename__ = 'volunteer_form'
+    id = Column(Integer, primary_key=True)
+    # ...
+    created_at = Column(DateTime, default=datetime.utcnow)
+    updated_at = Column(DateTime, default=datetime.utcnow, onupdate=datetime.utcnow)
+    site_id = Column(Integer, ForeignKey('site.id'))
+
+    site = relationship('Site', back_populates='volunteer_forms')
+
+
+class SponsorForm(db.Model):
+    __tablename__ = 'sponsor_form'
+    id = Column(Integer, primary_key=True)
+    # ...
+    created_at = Column(DateTime, default=datetime.utcnow)
+    updated_at = Column(DateTime, default=datetime.utcnow, onupdate=datetime.utcnow)
+    site_id = Column(Integer, ForeignKey('site.id'))
+
+    site = relationship('Site', back_populates='sponsor_forms')
+
+
+class SpeakerForm(db.Model):
+    __tablename__ = 'speaker_form'
+    id = Column(Integer, primary_key=True)
+    # ...
+    created_at = Column(DateTime, default=datetime.utcnow)
+    updated_at = Column(DateTime, default=datetime.utcnow, onupdate=datetime.utcnow)
+    site_id = Column(Integer, ForeignKey('site.id'))
+
+    site = relationship('Site', back_populates='speaker_forms')
+
+
+class Talk(db.Model):
+    __tablename__ = 'talk'
+    id = Column(Integer, primary_key=True)
+    title = Column(String)
+    content = Column(String)
+    video_url = Column(String)
+    slides_url = Column(String)
+    created_at = Column(DateTime, default=datetime.utcnow)
+    updated_at = Column(DateTime, default=datetime.utcnow, onupdate=datetime.utcnow)
+    slides_id = Column(Integer, ForeignKey('media.id'))
+    site_id = Column(Integer, ForeignKey('site.id'))
+
+    slides = relationship('Media', foreign_keys=[slides_id])
+    site = relationship('Site', back_populates='talks')
+    schedule_items = relationship('ScheduleItem', back_populates='talk')
+    talk_categories = relationship('TalkCategory', back_populates='talk')
+
+
+class Category(db.Model):
+    __tablename__ = 'category'
+    id = Column(Integer, primary_key=True)
+    name = Column(String)
+    created_at = Column(DateTime, default=datetime.utcnow)
+    updated_at = Column(DateTime, default=datetime.utcnow, onupdate=datetime.utcnow)
+
+    talk_categories = relationship('TalkCategory', back_populates='category')
+
+
+class TalkCategory(db.Model):
+    __tablename__ = 'talk_category'
+    id = Column(Integer, primary_key=True)
+    talk_id = Column(Integer, ForeignKey('talk.id'))
+    category_id = Column(Integer, ForeignKey('category.id'))
```

Comment:
We also need cascade deletion for relationship rows right?

Reply:
Updated.